README.md

```diff
@@ -47,6 +47,21 @@ brew link --force --overwrite protobuf241
 
 _For both Windows and Mac : To verify that installation was successful, `protoc --version` should display `2.4.1`_
 
+### Adding AEM dependent libraries ###
+
+#### Get workflow-console jar from AEM instance
+
+1. Ensure you have the FileVault tool (https://docs.adobe.com/docs/en/aem/6-1/develop/dev-tools/ht-vlttool.html) installed.
+2. Go to http://localhost:4502/system/console/bundles
+3. Search for workflow-console
+4. Notice `Bundle Location` and copy the path after jcrinstall: (/libs/cq/platform/install/)
+5. Open CRXDE Lite and add the copied path. You should see the name of the jar inside that path
+6. Now that the location is confirmed. Go to terminal and vlt down the jar. In my case this is what I used:
+
+`vlt --credentials admin:admin co --force http://localhost:4502/crx/-/jcr:root/libs/cq/platform/install`
+7. Copy the workflow-console jar (e.g. cq-workflow-console-5.8.2.jar) to the root directory (grabbit/grabbit/).
```

Comment:
Simpler:

1. Go to http://localhost:4502/system/console/bundles
2. Search for workflow-console
3. Notice `Bundle Location` and copy the path after jcrinstall: (e.g., `/libs/cq/platform/install/cq-workflow-console-5.8.2.jar`)
4. From the top project directory, simply use the Sling download component to grab the data.
For example
```
curl -u admin:admin -o cq-workflow-console-5.8.2.jar "http://localhost:4502/crx/de/download.jsp?path=/libs/cq/platform/install/cq-workflow-console-5.8.2.jar/jcr:content/jcr:data"
```